docker-compose.yml

```diff
@@ -164,8 +171,52 @@ services:
     image: prom/pushgateway:v1.3.0
     restart: always
 
+  # Analytics
+
+  mail:
+    image: bytemark/smtp
+    restart: always
```

Comment:
Unsurprisingly, this isn't working. We could perhaps live without email notifications from the service but the problem is that it gates the login:

<img width="596" alt="CleanShot 2021-01-18 at 11 39 18@2x" src="https://user-images.githubusercontent.com/65520/104905454-a98ad280-5982-11eb-8d43-bcb06ac87ef6.png">

Reply:
Mail logs:

```
  241 delivering 1l1Rsi-00003s-Pf
  241 R: dnslookup for sas@finestructure.co
  242 T: remote_smtp for sas@finestructure.co
  242 Connecting to mx.hover.com.cust.hostedemail.com [216.40.42.4]:25 ... connected
  242   SMTP<< 450 4.7.1 Client host rejected: cannot find your reverse hostname, [208.52.185.184]
  242   SMTP>> QUIT
  242   SMTP(close)>>
  242 LOG: MAIN
  242   H=mx.hover.com.cust.hostedemail.com [216.40.42.4]: SMTP error from remote mail server after initial connection: 450 4.7.1 Client host rejected: cannot find your reverse hostname, [208.52.185.184]
  241 LOG: MAIN
  241   == sas@finestructure.co R=dnslookup T=remote_smtp defer (0) H=mx.hover.com.cust.hostedemail.com [216.40.42.4]: SMTP error from remote mail server after initial connection: 450 4.7.1 Client host rejected: cannot find your reverse hostname, [208.52.185.184]
  243 LOG: MAIN
  243   <= hello@plausible.local H=spi_plausible_1.spi_default (spi-dev.swiftpackageindex.com) [172.19.0.18] P=esmtp S=1026
  243 LOG: smtp_connection MAIN
  243   SMTP connection from spi_plausible_1.spi_default (spi-dev.swiftpackageindex.com) [172.19.0.18] closed by QUIT
  244 Exim version 4.89 uid=101 gid=101 pid=244 D=40001
```

We'd need an authenticated SMTP that we can relay to and which allows us to connect from a container that doesn't support reverse IP lookup.

Reply:
There are a few providers that will give us a few emails/day for free on "developer" plans, which is all we'd ever need for this.

https://www.mailjet.com/pricing/ give us 200/day going to ~$10/mo after 6,000 emails/mo.
https://sendgrid.com/pricing/ give 100/day going ~$15/mo after 3,000 emails/mo

Both are reputable companies that maintain good sending reputation.

It's also possible that we'll want to expand this as we look at email functionality in the future, although we'd need to approach this carefully as email sending can get expensive quickly.

Should we configure a Mailjet account?

Reply:
https://www.sendinblue.com/pricing/ do 300/day and seem reputable from a bit of research, but I've not used them before. I've used both Mailjet and Sendgrid in previous projects.

Reply:
Same, but I think we can use a gmail account for sending. Trying that out right now!